bangc-ops/mlu_op.h

```diff
@@ -8485,6 +8486,7 @@ mluOpRoiawarePool3dForward(mluOpHandle_t handle,
  * @par Scale Limitation
  * - The value of \b boxes_num should be less than 65536.
  * - The value of \b channels should be less than 65536.
+ * - The value of \b max_pts_each_voxel should be no more than 98240.
```

Comment:
这两种说法二选一吧？

```suggestion
 * - The maximum value of \b max_pts_each_voxel is 98240.
```

或者

```suggestion
 * - The value of \b max_pts_each_voxel cannot be greater than 98240.
```

Reply:
选用第二种